specification/eventgrid/Azure.Messaging.EventGrid.SystemEvents/Microsoft.Communication/AzureCommunicationServices.tsp

```diff
@@ -36,13 +36,63 @@ model AcsUserDisconnectedEventData {
   userCommunicationIdentifier: CommunicationIdentifierModel;
 }
 
+/** Schema of the Data property of an EventGridEvent for a Microsoft.Communication.CallStarted event. */
+#suppress "@azure-tools/typespec-azure-core/composition-over-inheritance" "Maintain compatibility with existing models."
+model AcsCallStartedEventData
+  extends AcsCallingEventProperties {
+}
+
+/** Schema of the Data property of an EventGridEvent for a Microsoft.Communication.CallEnded event. */
+#suppress "@azure-tools/typespec-azure-core/composition-over-inheritance" "Maintain compatibility with existing models."
+model AcsCallEndedEventData
+  extends AcsCallingEventProperties {
+  /** The communication identifier of the user who was disconnected */
+  endedBy?: AcsCallingEventEndedByProperties;
+
+  /** The reason for the call ended event. */
+  reason?: AcsCallingEventReasonProperties;
+
+  /** Duration of the call in seconds. */
+  callDurationInSeconds?: float64;
+}
+
+/** Schema of the Data property of an EventGridEvent for a Microsoft.Communication.CallParticipantAdded event. */
+#suppress "@azure-tools/typespec-azure-core/composition-over-inheritance" "Maintain compatibility with existing models."
+model AcsCallParticipantAddedEventData
+  extends AcsCallParticipantEventProperties {
+}
+
+/** Schema of the Data property of an EventGridEvent for a Microsoft.Communication.CallParticipantRemoved event. */
+#suppress "@azure-tools/typespec-azure-core/composition-over-inheritance" "Maintain compatibility with existing models."
+model AcsCallParticipantRemovedEventData
+  extends AcsCallParticipantEventProperties {
```

Comment:
you missing reason and removedBy? for participantRemovedEvent
![{F1238126-15DA-4B73-848C-25F636E1E530}](https://github.com/user-attachments/assets/5f11e8b3-13d4-495d-828a-7649555c9f2d)